### DISCUSSION TITLE
Can the default themes be overridden whilst retaining dark/light mode switching?

### DISCUSSION THREAD

**Discussion by cbovis** [storybookjs/storybook#23514] [Help]
By default Storybook switches between light and dark themes based on my system preference. This is awesome however I'd like to override some of the theme defaults like brand image.

When overriding the default theme you need to specify a base which then ties the theme to either light or dark mode regardless of system setting. Is it possible to keep the automatic switching while overriding some aspects of both the light and dark themes?

**Comment by shilman** [comment]
Not currently but there's an RFC for that here https://github.com/storybookjs/storybook/discussions/23459
